feature-libs/product/configurators/rulebased/components/show-more/configurator-show-more.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="cx-show-more" *ngIf="text">
+  <div #textContent class="show-more-text">
+    {{ textToShow }}
```

Comment:
wrap this text into a separate `span` just for the sake of HTML correct semantics

Reply:
add